drivers/serial/Kconfig.intel_lw

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config UART_INTEL_LW
```

Comment:
Probably a good idea to make it a `menuconfig` so it can enclose the following config. Or else it would be very confusing to people using `menuconfig` as the next config appears without mentioning what that is for.

Reply:
Hi, @dcpleung. Thank you for the feedback. Done.